src/main/resources/db/migration/V20__behandle_sak_oppgave.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE behandle_sak_oppgave (
```

Comment:
Når vi får en anke, skal vi opprette oppgave for det?

Reply:
Nei